docs/source/integration-guide.rst

```diff
@@ -0,0 +1,189 @@
+Integration Guide
+=================
+
+This guide explains how to use ``astrolabe`` to add Atlas Planned Maintenance testing
+to your driver.
+
+
+--------
+Overview
+--------
+
+Driver projects looking to integrate this framework into their project codebase should follow the following steps:
+
+#. Implement a Workload Executor script for their driver project.
+#. Implement a Driver Installation script for their driver project.
+#. Update the Evergreen configuration of the ``drivers-atlas-testing`` project.
+
+
+We will walk through each of these steps in detail in the following sections.
+
+Setting up
+----------
+
+#. Start by cloning the ``drivers-atlas-testing`` repository and creating a new branch.
+#. Create a new directory for your driver under the ``drivers-atlas-testing/.evergreen`` directory.
+   It is recommended that drivers use the name of their language to name their folder.
+   Languages that have multiple drivers should use subdirectories named after the driver (e.g. for Python,
+   we should create ``.evergreen/python/pymongo`` and ``.evergreen/python/motor``). The path to the newly created
+   folder(s) will henceforth be referred to in this guide as ``DRIVER_DIRNAME``.
+
+----------------------------------------
+Step 1: Implementing a Workload Executor
+----------------------------------------
+
+Drivers must implement this standalone script in accordance with the specification, see :ref:`workload-executor-specification`.
```

Comment:
The referenced section describes the public API of the workload executor and also provides a pseudocode example (its basically the workload executor section of the specification)

Reply:
The implementation guide doesn't inform the user of the API required for the workload executor since that would duplicate information and necessitate its maintenance in two places.

Reply:
Ok, thanks!